scripts/validate-security.sh

```
@@ -0,0 +1,176 @@
+    npm audit
+    ((WARNINGS++))
+  fi
+else
+  echo -e "${RED}❌ npm auditの実行に失敗しました${NC}"
+  ((ERRORS++))
+fi
+echo ""
+
+# 3. HTTPS通信の確認
+echo "📋 3. HTTPS通信の使用確認"
+HTTP_COUNT=$(grep -r "http://" src/ --include="*.js" | grep -v "https://" | wc -l | tr -d ' ')
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)